lambda/snapshots_tool_utils.py

```diff
@@ -362,3 +364,98 @@ def copy_remote(snapshot_identifier, snapshot_object):
             CopyTags = True)
 
     return response
+
+
+def get_all_automated_snapshots(client):
+    global _AUTOMATED_BACKUP_LIST
```

Comment:
Why does this have to be a global variable?

Reply:
caching this variable to reduce api calls